website/docs/r/tenant_policies_route_map_policy_multicast.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "mso"
+page_title: "MSO: mso_tenant_policies_route_map_policy_multicast"
+sidebar_current: "docs-mso-resource-tenant_policies_route_map_policy_multicast"
+description: |-
+  Manages Route Map Policies for Multicast on Cisco Nexus Dashboard Orchestrator (NDO)
+---
+
+
+
+# mso_tenant_policies_route_map_policy_multicast #
+
+Manages Route Map Policies for Multicast on Cisco Nexus Dashboard Orchestrator (NDO). This resource is supported in NDO v4.4(1) or higher.
+
+## GUI Information ##
+
+* `Location` - Manage -> Tenant Template -> Tenant Policies -> Route Map Policy for Multicast for Multicast
```

Comment:
`Location` - Manage -> Tenant Template -> Tenant Policies -> Route Map Policy for Multicast